src/common.jl

```diff
@@ -194,10 +213,11 @@ const XPRS_TREEMEMORYLIMIT = 8242
 const XPRS_TREECOMPRESSION = 8243
 const XPRS_TREEDIAGNOSTICS = 8244
 const XPRS_MAXGLOBALFILESIZE = 8245
-const XPRS_TEMPBOUNDS = 8250
+const XPRS_REPAIRINFEASMAXTIME = 8250
```

Comment:
maybe leave both witha comment next to the older one sayin its deprecaded

Reply:
ok